rule_packages/c/Strings3.json

```diff
@@ -12,6 +12,7 @@
           "precision": "very-high",
```

Comment:
when the rule is shared between c and c++, doesnt it need to have the shared implementation name portion in the package description file for both c/c++?

for example [like this](https://github.com/search?q=repo%3Agithub%2Fcodeql-coding-standards+IncludeGuardsNotUsed+language%3AJSON&type=code&l=JSON)?